packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -75,27 +76,42 @@ export const start = async (zcf, privateArgs) => {
 
   assertElectorateMatches(electorateParamManager, governedTerms);
 
+  /** For temporary staging of newly minted tokens */
+  const { zcfSeat: mintSeat } = zcf.makeEmptySeatKit();
   const { zcfSeat: rewardPoolSeat } = zcf.makeEmptySeatKit();
 
   /**
    * We provide an easy way for the vaultManager to add rewards to
    * the rewardPoolSeat, without directly exposing the rewardPoolSeat to them.
    *
-   * @type {ReallocateWithFee}
+   * @type {MintAndReallocate}
    */
-  const reallocateWithFee = (fee, fromSeat, otherSeat = undefined) => {
-    rewardPoolSeat.incrementBy(
-      fromSeat.decrementBy(
-        harden({
-          RUN: fee,
-        }),
-      ),
-    );
-    if (otherSeat !== undefined) {
-      zcf.reallocate(rewardPoolSeat, fromSeat, otherSeat);
-    } else {
-      zcf.reallocate(rewardPoolSeat, fromSeat);
+  const mintAndReallocate = (toMint, fee, seat, ...otherSeats) => {
+    const wanted = AmountMath.subtract(toMint, fee);
```

Comment:
"want" is a proposal concept which is not the concern of this function. consider `proceeds`

Reply:
In all cases, we are minting because an offer wanted it, so aligns well when reading its usage. But I shall consider your alternative word.